cmd/snap/cmd_find.go

```diff
@@ -100,6 +101,10 @@ func (x *cmdFind) Execute(args []string) error {
 		return ErrExtraArgs
 	}
 
+	if x.Positional.Query == "" {
+		return errors.New(i18n.G("you need to specify a query. For example: snap find hello-world"))
```

Comment:
i shall take this opportunity and make a unifying pass at all the little suggestions